blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/TestIO.java

```diff
@@ -0,0 +1,24 @@
+package com.rackspacecloud.blueflood.io;
+
+/**
+ * Utility class to provide some test methods existing tests use.
+ */
+public interface TestIO {
```

Comment:
My thought was to make a helper class that might be usable in other contexts as well, not just for MetadataIO.